src/main/kotlin/com/example/iotManagementDevice/IotManagementDeviceApplication.kt

```diff
@@ -1,11 +1,12 @@
-package com.example.iot_management_device
+package com.example.iotManagementDevice
```

Comment:
```suggestion
package com.example.iotmanagementdevice
```
or think about shorter name